packages/react/src/components/header/components/navigationLink/NavigationLink.module.scss

```diff
@@ -0,0 +1,28 @@
+.navigationLink.navigationLink {
+  --link-visited-color: none;
+  --link-color: var(--color-black-90);
+  display: flex;
+  line-height: var(--lineheight-l);
+  text-decoration: none;
+  word-wrap: break-word;
+}
```

Comment:
Hmm, now that we have the navigationLinkWrapper, could we move this inside that? The variables could be left here though (so that the variables would be at the start of the file)

Reply:
Donzo